_episodes/00-background.md

```diff
@@ -5,22 +5,33 @@ exercises: 0
 questions:
     - "What is the Ocean Tracking Network?"
     - "How does your local telemetry network interact with OTN?"
-    - "What methods of data analysis will be covered?"
 keypoints:
     - "OTN is a global organization."
+    - "OTN supports research in ways both technical and data related"
     - "More informatin is available on the Data Portal"
 
 ---
-## Intro to OTN
+# Intro to OTN
 
-The Ocean Tracking Network (OTN) supports global telemetry research by providing training, equipment, and data infrastructure to our large network of partners. 
+The Ocean Tracking Network (OTN) supports global telemetry research by providing training, equipment, and data infrastructure to our large network of partners.
+
+## OTN Field Technicians
```

Comment:
words are hard, change anything you want here, please!

Reply:
edits added!